tools/mmp/driver.cs

```diff
@@ -800,6 +808,14 @@ static void Pack (IList<string> unprocessed)
 			if (App.LinkMode != LinkMode.All && App.RuntimeOptions != null)
 				App.RuntimeOptions.Write (App.AppDirectory);
 
+			if (aot.IsAOT) {
+				if (!IsUnified || !Is64Bit)
+					throw new MonoMacException (98, true, "AOT compilation is only available on Unified 64-bit", ret);
```

Comment:
Agreed. I assume it won't work, and will have to ship another, which isn't hard. The error/check is to make sure that we don't forget and ship brokeness. I'll remove it when we decide/fix it one way or the either.

Made this trello card - https://trello.com/c/1HLne9yA/315-aot-32-bit